pkg/traces/promsdprocessor/prom_sd_processor.go

```diff
@@ -16,13 +16,19 @@ import (
 	"github.com/prometheus/prometheus/discovery/targetgroup"
 	"github.com/prometheus/prometheus/pkg/labels"
 	"github.com/prometheus/prometheus/pkg/relabel"
+	"go.opentelemetry.io/collector/client"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/component/componenterror"
 	"go.opentelemetry.io/collector/consumer"
 	"go.opentelemetry.io/collector/model/pdata"
 	semconv "go.opentelemetry.io/collector/model/semconv/v1.6.1"
 )
 
+const (
+	ipTagName    = "ip"
+	k8sIPTagName = "k8s.pod.ip"
+)
```

Comment:
I think these can be deleted now that we have the pod association labels?

Reply:
Done, thanks!